ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -100,6 +100,11 @@ def dim(self):
     def abs(self):
         return paddle_frontend.abs(self)
 
```

Comment:
Please add this here: `@with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")`